boudicca.base/publisher-event-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,48 +1,439 @@
+<style>
+    .event {
```

Comment:
please move this back to events.hbs, because event.hbs is rendered in a loop, so you now have this css block for each event we show :D

Reply:
yep, done.